docs/worker-node/using-wn.md

```diff
@@ -10,6 +9,16 @@ The OSG provides no scientific software dependencies or software build tools on
 
 If you would like to test the minimal OS environment that jobs can expect, you can test out your scientific software in [the OSG Docker image](https://hub.docker.com/r/opensciencegrid/osg-wn/).
 
+Suggested Hardware Requirements on Worker Nodes
+-----------------------------------------------
+| Hardware               | Minimum | Recommended                         | Notes                                             |
+|:-----------------------|:--------|:----------------------|:--------------------------------------------------|
+|Core per pilot                   |  1      |8                      | Dependends on the supported users. The total core count on every node in the cluster must be divisible by core per pilot.|
```

Comment:
Huh, is the second sentence true? I didn't know that.

```suggestion
|Core per pilot                   |  1      |8                      | Depends on the supported VOs. The total core count on every node in the cluster must be divisible by core per pilot.|
```

Reply:
@brianhlin of course otherwise you end up in suggestions you don't want to be. Either cores never utilized or a continuously fragmented pool